source/math/FloatMatrix.ooc

```diff
@@ -24,25 +25,26 @@ FloatMatrix : cover {
 	dimensions ::= this _dimensions
 	width ::= this _dimensions width
 	height ::= this _dimensions height
-	elements: Float[]
+	_elements: OwnedBuffer
+	elements ::= this _elements pointer as Float*
```

Comment:
So I tried building in release mode and now the function is called more times than the next 7 or so functions combined. It's easily fixed, though - whenever you need `this elements` more than once in a function (especially before a loop), store it in a variable
```ooc
buffer := this elements
for (i in 0 .. width)
    buffer[x * i]
```
etc.